source/usage-examples/findOne.txt

```diff
@@ -3,3 +3,99 @@ Find a Document
 ===============
 
 .. default-domain:: mongodb
+
+You can retrieve a single document from your collection with the
+``Collection.FindOne()`` method. 
+
+..
+   method
+
+The ``FindOne()`` method takes a filter document. Calling the ``FindOne()``
+method retrieves a document from your MongoDB instance that matches your
+filter document. If you pass the ``FindOne()`` method an empty filter document,
+MongoDB considers your entire collection when retrieving your document. For more
+information on documents in the MongoDB Go driver, see our
+:doc:`guide on data formats </fundamentals/data-formats>`.
+
+You must pass the ``FindOne()`` method a ``context.Context`` interface type
+value. For more information on the ``context.Context`` interface and the
+``context`` package, see our :doc:`guide on context </fundamentals/context>`.
+
+..
+   options
+
+You can specify options that change the behavior of the ``FindOne()`` method.
+One example of an option is **sort**, which instructs your MongoDB instance to
+order the matching documents of your collection before it retrieves the
+first document. Another example of an option is a **projection**, which
+specifies or restricts the fields in your returned document. You can specify
+options by passing the ``FindOne()`` method a pointer to a ``FindOneOptions`` struct.  
+You can configure a ``FindOneOptions`` struct by using setter methods such as the
+``FindOneOptions.SetProjection()`` method. For more information on the sort operation,
```

Comment:
nit: should this be sort operation or sort option? (same for projection) I'm not sure what the docs convention is.

Reply:
Thanks for the note. As the linked page will discuss sort generally in the context of the Go driver (similarly [to this page](https://docs-mongodbcom-staging.corp.mongodb.com/d7956eb/java/docsworker-xlarge/DOCSP-9572-Logging/fundamentals/crud/read-operations/sort/) for the Java driver docs), I think either term is appropriate.

After rereading the paragraph, I can see how "sort option" may be clearer for readers as sorting was discussed earlier in the context of options.

Edit: